src/main.rs

```diff
@@ -56,20 +63,24 @@ pub struct Config {
     prometheus_listen: Option<String>,
 }
 
-/// Parse a timestamp given as a config option
-fn parse_timestamp(stamp: &str) -> Result<OffsetDateTime, &'static str> {
-    OffsetDateTime::parse(stamp, &Rfc3339).map_err(|_| "Try something like '2023-05-15T04:30:00Z'.")
-}
-
 /// Initialize an axum::Router for our web service
 /// Having this as a separate function makes testing easier.
 fn app(oprf_state: OPRFState) -> Router {
     Router::new()
         // Friendly default route to identify the site
         .route("/", get(|| async { "STAR randomness server\n" }))
-        // Main endpoints
-        .route("/randomness", post(handler::randomness))
-        .route("/info", get(handler::info))
+        // Endpoints for all instances
+        .route(
+            "/instances/:instance/randomness",
+            post(handler::specific_instance_randomness),
+        )
+        .route(
+            "/instances/:instance/info",
+            get(handler::specific_instance_info),
```

Comment:
This isn't discoverable. What about an `instances` route that returns a list of the available instance names, or a combined info doc for all the instances at once?

Reply:
good ideas. decided to go with the `instances` route, to avoid breaking changes.